lib/internal/test_runner/utils.js

```diff
@@ -222,6 +224,48 @@ function parseCommandLine() {
   return globalTestOptions;
 }
 
+function coverageThreshold(coverage, color = white) {
+  coverage = NumberPrototypeToFixed(coverage, 2);
+  if (color !== white) {
+    if (coverage > 90) return `${green}${coverage}${color}`;
+    if (coverage < 50) return `${red}${coverage}${color}`;
+  }
+  return coverage;
+}
+
+function getCoverageReport(pad, summary, symbol = '# ', color = white) {
```

Comment:
Please remove the default values for `symbol` and `color`. The TAP reporter can pass them explicitly. Also, instead of using `white` for the TAP reporter, please use the empty string (the value of `white` if colors are not supported).